taxcalc/functions.py

```diff
@@ -154,6 +194,10 @@ def AGI(_ymod1, c02500, c02700, e02615, c02900, e00100, e02500, XTOT,
 
     if DSI:
         XTOT = 0
+    # elif puf:
```

Comment:
This part was moved to a new PR. It has been confirmed to affect the tests against Internet TAXSIM.